include/seqan3/alignment/configuration/align_config_parallel.hpp

```diff
@@ -0,0 +1,42 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides seqan3::align_cfg::parallel configuration.
+ * \author Lydia Buntrock <lydia.buntrock AT fu-berlin.de>
+ * \author Jörg Winkler <j.winkler AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/alignment/configuration/detail.hpp>
+#include <seqan3/core/algorithm/pipeable_config_element.hpp>
+
+namespace seqan3::align_cfg
+{
+/*!\brief Sets a flag whether the alignment computation should be performed in parallel.
```

Comment:
Enables the parallel execution of the alignment algorithm if possible for the given configuration.